### PR TITLE
edit mer-cursor

### PR DESCRIPTION
**Pull request by Yutyo**
mer-cursor bug fix. (you have done it, so well, you would approve it)

### CODE REVIEW

File: config/base/software.nix [hunk 1]
```diff
@@ -14,6 +14,9 @@ with lib;
     git
     curl
     wget
+    vim
+    gcc
```
Comment (mkg20001) [inline config/base/software.nix]: unsure if we need this to be part of the default set

I'm thinking maybe we can move atom & the dev stuff to a dev meta-package that users can optionally install

Comment (mkg20001) [reply config/base/software.nix]: Can we split this into anther PR?

Comment (Yutyo) [reply config/base/software.nix]: vim and gcc aren't that heavy tho.

Comment (mkg20001) [reply config/base/software.nix]: My opinion is less based on the size, but rather on usefulness

Compared to this, would doing `nix-env -iA meros.devTools` to get the devtools be an idea?

**Pull request merged**